### PR TITLE
camelcase service in all work filter

### PR DESCRIPTION
**Pull request by Surekha-HMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-7336

### Change description ###

camelcase service in all work filter

### CODE REVIEW

File: api/.env.defaults [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NODE_CONFIG_DIR=../config
 ALLOW_CONFIG_MUTATIONS=1
-NODE_CONFIG_ENV=development
+NODE_CONFIG_ENV=demo
```
Comment (udaydenduluri33) [inline api/.env.defaults:3]: Please remove this

**Review by DavidJayakumar — APPROVED** [review]
Looks good

File: src/work-allocation/components/task-list-filter/task-list-filter.component.ts [hunk 2]
```diff
@@ -435,4 +435,12 @@ export class TaskListFilterComponent implements OnInit, OnDestroy {
       }, 0);
     }
   }
+
+  private titleCase(string){
```
Comment (udaydenduluri33) [inline src/work-allocation/components/task-list-filter/task-list-filter.component.ts:439]: Can you add a comment here to explain what is that you doing?

**Comment by Surekha-HMCTS** [comment]
Closing this, as another PR is created against r4 - https://github.com/hmcts/rpx-xui-webapp/pull/2849

**Pull request closed without merging**